completions/bash/multipass

```diff
@@ -21,25 +21,105 @@ _multipass_complete()
         local cmd="multipass list --format=csv --no-ipv4"
         [ -n "$state" ] && cmd="$cmd | \awk -F',' '\$2 == \"$state\"'"
 
-        local instances=$( \eval $cmd | \grep -Ev '(\+--|Name)' | \cut -d',' -f 1 )
+        local retval=$( \eval $cmd | \grep -Ev '(\+--|Name)' | \cut -d',' -f 1 )
+
+        for instance in $retval; do
+            instances="${instances} ${instance}"
+        done
```

Comment:
This is only to replace newlines with spaces, right? Wouldn't this work?

```
instances=${retval//?($'\r')$'\n'/ }
```

In any case, it is repeated in different places, so I would ask to extract into a function and call that (e.g. `_flatten_lines`).